pkg/kubelet/server_test.go

```diff
@@ -931,6 +936,237 @@ func TestServeExecInContainer(t *testing.T) {
 	}
 }
 
+// TODO: largely cloned from TestServeExecContainer, refactor and re-use code
+func TestServeAttachContainer(t *testing.T) {
```

Comment:
Please take a look at https://github.com/GoogleCloudPlatform/kubernetes/pull/10837-- one of you will have to rebase